scripts/buildLogos.js

```diff
@@ -40,6 +44,12 @@ async function buildLogos() {
 
   // eslint-disable-next-line no-restricted-globals,compat/compat,promise/no-native
   await Promise.all(logoPromises);
+
+  if (shouldCommit) {
+    await $`git add .`;
+    await $`git commit -am "feat: adding ${outdir}"`;
```

Comment:
I like the idea of using "Generate CDN packages" as the commit message here to maintain parity with SPB and SDK-Release but I don't feel strongly about this.